examples/supplementary/format_concat_args/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "format_concat_args"
+version = "0.1.0"
```

```suggestion
version = "4.1.0"
```
To match the versions used elsewhere.